docs/user-guide/install-zowe-smpe.md

```diff
@@ -712,6 +712,33 @@ IEF257I ZWE3ALOC ALLOCD ALLOCD AZWEZFS - SPACE REQUESTED NOT AVAILABLE
 IEF272I ZWE3ALOC ALLOCD ALLOCD - STEP WAS NOT EXECUTED.                    
 ```
 Uncomment the `VOL=SER=&...` control statements and refer to the comments at the start of the JCL job for related necessary changes.
+### Prepare the Zowe SMP/E Installation Jobs with Velocity Template Language (VTL) Command-line Interface or Python Script (Optional)
+The Velocity Template Language (VTL) Command-line Interface and Python Script simplifies the Zowe SMP/E installation jobs customization using the YAML file.  To install and use the Velocity Template Language (VTL) Command-line Interface, refer to https://github.com/plavjanik/vtl-cli.
+
+
+To prepare the custom Zowe SMP/E installation jobs, edit the YAML file and execute the VTL command or python script. The YAML and the corresponding VTL files are located in the ZOWE SMP/E repository. Perform the following steps to prepare the customized jobs:
+1.	Download the YMAL and VTL files to your system from …….
+2.	Open the YAML file, replace the default variable values with the custom values to meet your site requirements. 
+3.	Generate the custom JCLs with one of the following options:
+    - **VTL Command**
+
```

Comment:
**General comment:** Add an extra line between two separate sentences. Otherwise, the sentences will show in one line in Markdown. It's a Markdown formatting issue. For example，add an extra line after `Generates a single custom JCL in the specified location.` :)